schema/spacewalk/susemanager-schema.changes

```diff
@@ -1,3 +1,4 @@
+- Add support for Alibaba Cloud Linux 2
```

Comment:
Since you are adding EPEL8 as well, please mention it at the changelog. You can just add a new entry if needed.

Reply:
Added